browser/src/Config.ts

```diff
@@ -86,14 +86,33 @@ Performance.mark("Config.load.start")
 
 export const userJsConfig = path.join(getUserFolder(), "config.js")
 
-let userConfig = {}
-let userRuntimeConfig = {}
-
-if (fs.existsSync(userJsConfig)) {
-    userRuntimeConfig = global["require"](userJsConfig) // tslint:disable-line no-string-literal
+let Config = {}
+
+function applyConfig(): void {
+    let userRuntimeConfig = {}
+    if (fs.existsSync(userJsConfig)) {
+        userRuntimeConfig = global["require"](userJsConfig) // tslint:disable-line no-string-literal
+    }
+    Config = { ...DefaultConfig, ...DefaultPlatformConfig, ...userRuntimeConfig }
 }
 
-const Config = { ...DefaultConfig, ...DefaultPlatformConfig, ...userConfig, ...userRuntimeConfig }
+applyConfig()
+// use watch() on the directory rather than on config.js because it watches
```

Comment:
Thanks for the detailed comment here! If we are consistently getting double events, we could look at `debounce`'ing the event: https://lodash.com/docs/4.17.4#debounce